### ISSUE TITLE
AttributeError: 'gi.repository.GLib' object has no attribute 'USER_DIRECTORY_HOME'

### ISSUE THREAD

**Issue by Anonymous941** [giantpinkrobots/varia#125]
When using the Flatpak version, it fails to launch:
```plain
/bin/bash: warning: setlocale: LC_ALL: cannot change locale (en_US_custom.UTF-8)

(process:4): Gtk-WARNING **: 14:39:39.655: Locale not supported by C library.
        Using the fallback 'C' locale.
Traceback (most recent call last):
  File "/app/share/varia/varia/variamain.py", line 379, in main
    if (os.path.exists(GLib.get_user_special_dir(GLib.USER_DIRECTORY_DOWNLOAD))):
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen genericpath>", line 19, in exists
TypeError: stat: path should be string, bytes, os.PathLike or integer, not NoneType

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/app/share/varia/../../bin/varia-py.py", line 36, in <module>
    sys.exit(main(VERSION, aria2cexec))
             ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/share/varia/varia/variamain.py", line 384, in main
    download_directory = GLib.get_user_special_dir(GLib.USER_DIRECTORY_HOME)
                                                   ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/gi/overrides/__init__.py", line 31, in __getattr__
    return getattr(self._introspection_module, name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/gi/module.py", line 130, in __getattr__
    raise AttributeError("%r object has no attribute %r" % (
AttributeError: 'gi.repository.GLib' object has no attribute 'USER_DIRECTORY_HOME'
```

**Comment by N0tACyb0rg** [comment]
This was fixed in #108, a new release just needs to be released for it to be fixed.

**Comment by Anonymous941** [comment]
Got it, sorry for the duplicate
